src/analysis/retail/priest/discipline/modules/spells/PowerWordShield.tsx

```diff
@@ -176,11 +196,12 @@ class PowerWordShield extends Analyzer {
   }
 
   onPenanceHeal(event: HealEvent) {
-    this.shieldOfAbsolutionValue += event.amount * SHIELD_OF_ABSOLUTION_MULTIPLIER_HEALING;
+    this.shieldOfAbsolutionValue +=
+      (event.amount + (event.overheal || 0)) * SHIELD_OF_ABSOLUTION_MULTIPLIER_HEALING;
   }
 
   onPenanceDamage(event: DamageEvent) {
-    this.shieldOfAbsolutionValue += event.amount;
+    this.shieldOfAbsolutionValue += event.amount * SHIELD_OF_ABSOLUTION_MULTIPLIER_DAMAGE;
```

Comment:
This is placeholder as while the tooltips for the 4p are nerfed, the nerf is actually not yet implemented